test/controllers/issues_controller_test.rb

```diff
@@ -18,4 +18,57 @@ class IssuesControllerTest < ActionDispatch::IntegrationTest
     assert_equal test_line_id, issue.line_onestop_id
     assert_response :success
   end
+
+  test "does not allow for visitor to create issue" do
+    test_stop_id = "1"
+    test_line_id = "1"
+    get new_issue_url, params: { stop_id: test_stop_id, line_id: test_line_id }
+
+    assert_response :found
+  end
+
+  test "get index page for issues" do
+    sign_in users(:one)
+
+    get issues_path
+    assert_response :success
+  end
+
+  test "post create issues" do
```

Comment:
I'd change this verbiage to be clearer to what this test is verifying, perhaps: "can create issue"